test/.clang-format

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
you really should not sneak it as part of a completely different commit, but ok

Reply:
Ack

Reply:
made into a separate commit.